### PR TITLE
Footer changes: Solves issue #17

### PR DESCRIPTION
**Pull request by KartikChinda**
This is how the footer looks now: 
image

### CODE REVIEW

**Comment by KartikChinda** [comment]
@clebsonf please check this PR once and merge if it looks good to you!

**Comment by clebsonf** [comment]
The changes break the responsive layout for larger screens :/

**Comment by clebsonf** [comment]
image

File: src/components/footer/index.jsx [hunk 1]
```diff
@@ -1,21 +1,79 @@
 import Link from "next/link"
 import React from "react"
 
+
 export default function Footer() {
     return (
-        <footer className="sticky top-[100vh] p-4 h-[20vh] bg-white md:p-8 lg:p-10 dark:bg-gray-800">
-            <div className="mx-auto max-w-screen-xl text-center">
-                <p className="my-6 text-gray-500 dark:text-gray-400">
+        <footer class="text-center bg-gray-800 text-white">
```
Comment (clebsonf) [inline src/components/footer/index.jsx:7]: the footer must have a light and dark version, as before.

It also needs to get back to being responsive.

Other than that, it was a great job. I put some reference links, take a look if possible

**Review by clebsonf — CHANGES_REQUESTED** [review]

File: src/components/footer/index.jsx (hunk 1 above)
Comment (KartikChinda) [reply src/components/footer/index.jsx]: I've made another commit with changes to the file and also checked the responsiveness from the dev tools (responsiveness isnt my strong suit haha.) The light and dark versions have also been added. I hope it is good to go now?

**Review by clebsonf — APPROVED** [review]

File: src/components/footer/index.jsx (hunk 1 above)
Comment (clebsonf) [reply src/components/footer/index.jsx]: amazing!

**Pull request merged**